app/models/event.rb

```diff
@@ -1,23 +1,10 @@
 class Event < ApplicationRecord
-  belongs_to :move
+  belongs_to :entity, polymorphic: true
```

Comment:
I understand we need a generic name here,
but what actually entity is? there is not a model for it, if I full search in the project you get tens of result and it's difficult to have an idea about this `entity`

Reply:
agreed to rename to `related_entity`

Reply:
Looks like they're events related to either moves or journeys for now - and potentially other things in future. It's unfortunate that `event` is a widely used term but not sure what else would make sense here.

Reply:
`related_entity`, or `event_entity` are better name, at least they are not widely used.
As agree a new test helps to clarify that this entity can be

Reply:
These events will definitely relate to `moves` and journeys` and will likely relate to other things (people etc) in the future.
@smoothcontract is `related_entity` cleaner than `entity` ? It will be easier to search in the codebase.

Reply:
I don't mind `entity` at all - my comments were related to the overall name of the model (think I misunderstood @alexdesi original comment!). For polymorphic models, the normal Railsy convention is to somehow shoehorn `..able` on the end so that would make this `belongs_to :eventable, polymorphic: true`. Thoughts on that?

Reply:
yep, happy with `eventable`, and I will also rename `entity` to `event_entity` to make searching for it easier

Reply:
I think two things got confused here. I was suggesting leaving the model name as `Event`, with the `entity` attribute renamed to `eventable`.

Reply:
ok, we are on the same page now: `entity_id` --> `eventable_id`, `entity_type` --> `eventable_type`